tools/buildFactory.ts

```diff
@@ -137,7 +139,9 @@ export function gulpFactory(
     cleanStream,
     merge([tsBuildResult.dts, jsBuildResult])
```

Comment:
This method creates a source map only for js files.
```typescript
    merge([
      tsBuildResult.dts
        .pipe(filter(['**/*.ts', '!**/*.macro.d.ts'])),
      jsBuildResult
        .pipe(filter(['**/*.js', '!**/*.macro.js']))
        .pipe(sourceMap ? sourcemaps.write('.') : tap(() => { /* */ }))
    ])
```

Reply:
Thanks!